web/html/src/manager/maintenance/calendar/web-calendar.tsx

```diff
@@ -24,6 +25,15 @@ type Event = {
   title: string;
 }
 
+// TODO: Add more colors
```

Comment:
I would recommend picking something like the LCH color space and generating a long list of colors with a comment on how to add more colors if needed later. See https://darker.ink/writings/Towards-richer-colors-on-the-Web#consistent-and-predictable-colors for background on why LCH is a good choice for this. Could also pick up a library for this and generate them on the fly, but I prefer the static pre-built colors variant.